src/components/footer/www/footer.jsx

```diff
@@ -11,6 +11,21 @@ const frameless = require('../../../lib/frameless');
 
 require('./footer.scss');
 
+const interwikiMapping = {
+    de: 'https://de.scratch-wiki.info',
+    ru: 'https://ru.scratch-wiki.info/',
+    nl: 'https://nl.scratch-wiki.info/',
+    id: 'https://id.scratch-wiki.info/',
+    ja: 'https://ja.scratch-wiki.info/',
+    hu: 'https://hu.scratch-wiki.info/',
+    fr: 'https://fr.scratch-wiki.info/'
+};
+var wikiLink = null;
+if (this.props.intl.locale in interwiki_mapping) {
```

Comment:
Forgot to change this varname
```suggestion
if (this.props.intl.locale in interwikiMapping) {
```